html/components/plt-input.js

```diff
@@ -727,4 +727,41 @@ function prepareAnnotationEditor(teamId) {
 		}
 	}
 }
+
+var optionsDialog;
+
+function openOptionsDialog() {
+	optionsDialog.dialog('open');
+}
+
+function prepareOptionsDialog(teamId) {
+	var table = $('<table>').appendTo($('#OptionsDialog'));
+	$('<tr>').append($('<th>').text('Select Team')).appendTo(table);
+	$.each( [ '1', '2' ], function() {
+		var tId = String(this);
+		var row = $('<tr>').addClass('selectTeam'+tId).appendTo(table);
+		$('<td>').append($('<button>').addClass('name').toggleClass('selected', tId === teamId).button().click(function() {
+			if (tId !== teamId) {
+				window.location.href = window.location.href.replace( /[\?#].*|$/, '?team='+tId );
+			} else {
+				optionsDialog.dialog('close');
+			}
+		})).appendTo(row);
+	});
+
+	WS.Register(['ScoreBoard.Team(*).Name', 'ScoreBoard.Team(*).AlternateName(operator)'], function(k, v) {
+		var displayName = 'Team ' + k.Team + ': ' + WS.state['ScoreBoard.Team('+k.Team+').Name'];
+		var altName = WS.state['ScoreBoard.Team('+k.Team+').AlternateName(operator)'];
+		if (altName != null) { displayName = displayName + ' / ' + altName; }
```

Comment:
We only ever display one name, so this should be the operator falling back to the team name as we have elsewhere.

Reply:
That was an intentional diversion from the usual behaviour due to a feature request on FB a while ago that pointed out a potential problem when operator names are colours and tablets from multiple tracks are picked up from the same spot.